target/linux/generic/config-6.12

```diff
@@ -2070,6 +2103,7 @@ CONFIG_FSNOTIFY=y
 # CONFIG_FUNCTION_TRACER is not set
 # CONFIG_FUN_ETH is not set
 # CONFIG_FUSE_FS is not set
+CONFIG_FUSE_PASSTHROUGH=y
```

Comment:
This should be disabled by default. It's depend on `CONFIG_FUSE_FS`.
https://elixir.bootlin.com/linux/v6.12.19/source/fs/fuse/Kconfig#L56

Reply:
Ok, I'll do it in the next push.